src/components/Step1.tsx

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import { allCountries } from "./country";
+import {BsFillExclamationSquareFill}  from 'react-icons/bs'
+
+const validationSchema = yup.object().shape({
+  username: yup
+    .string()
+    .required("Username is required")
+    .min(4, "Username must be at least 4 characters")
+    .max(12, "Username must be at most 12 characters"),
+  email: yup
+    .string()
+    .required("Email is required")
+    .matches(
+      /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i,
+      "Please enter a valid email address"
+    ),
+  phoneNumber: yup
+    .string()
+    .required("Phone Number is required")
+    .matches(
+      /^(?:\+?\d{1,3}\s?)?(?:\d{10}|\d{3}[-\.\s]?\d{3}[-\.\s]?\d{4})$/,
+      "Invalid phone number"
+    ),
+  country: yup
+    .string()
+    .required("Country is required")
+    .matches(
+      /^(?!.*\bSelect\b)/, // Use negative lookahead to ensure "SELECT" is not present
+      "Country is required"
+    ),
```

Comment:
Create a schema file under Validation, move the schema there and import here,

Reply:
done